car_rental/demo/sale_order_post.xml

```diff
@@ -0,0 +1,143 @@
+<odoo>
+    <function model="sale.order" name="action_confirm">
+        <value eval="[
+            ref('car_rental.sale_order_1'),
+            ref('car_rental.sale_order_2'),
+            ref('car_rental.sale_order_3'),
+            ref('car_rental.sale_order_4'),
+            ref('car_rental.sale_order_7')]"/>
+    </function>
+
+<!--    Pickup product so1-->
+    <record id="car_rental.rental_order_wizard_so1_pickup" model="rental.order.wizard">
```

Comment:
Wouldn't it make more sense to define `qty_delivered` and `qty_returned` directly on the SO, instead of these wizard data?

Reply:
We prefer standard flow so if any other fields are updated then we no need to take care of it.
it's good for the maintenance of code.

Reply:
Ok, yes, it makes sense.